components/__snapshots__/quick_input.test.jsx.snap

```diff
@@ -0,0 +1,199 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
I normally prefer to avoid snapshots, but in this case it seemed onerous to bind the tests to a particular prop deep in the rendered hierarchy. So I verified the expected text changes, and then recorded them as the full snapshot.